specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ConnectedEnvironments.json

```diff
@@ -403,6 +403,15 @@
             "staticIp": {
               "description": "Static IP of the connectedEnvironment",
               "type": "string"
+            },
+            "daprAIConnectionString": {
+              "type": "string",
+              "description": "Application Insights connection string used by Dapr to export Service to Service communication telemetry",
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ],
+              "x-ms-secret": true
```

Comment:
The `read` mutability conflicts with `secret: true`. The mutability should be `[create, update]`?

Reply:
if so, only need set `secret: true`